webpack/JobWizard/steps/HostsAndInputs/SelectGQL.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { SelectVariant } from '@patternfly/react-core';
+import {
+  useForemanOrganization,
+  useForemanLocation,
+} from 'foremanReact/Root/Context/ForemanContext';
+import { HOSTS, HOST_GROUPS, dataName } from '../../JobWizardConstants';
+import { SearchSelect } from '../form/SearchSelect';
+
+export const queries = {
+  [HOSTS]: gql`
```

Comment:
Literal tag is always an option, it is also possible to have [queries in separate files](https://github.com/theforeman/foreman_openscap/blob/master/webpack/graphql/queries/hostgroups.gql) and [importing them](https://github.com/theforeman/foreman_openscap/blob/master/webpack/routes/OvalContents/OvalContentsIndex/OvalContentsIndex.js#L12).